src/main/kotlin/com/ampnet/auditornode/scriptapi/Http.kt

```diff
@@ -0,0 +1,27 @@
+package com.ampnet.auditornode.scriptapi
+
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.graalvm.polyglot.HostAccess.Export
+
+object Http {
+
+    private val client = OkHttpClient()
+
+    @Export
+    @JvmStatic
+    fun request(method: String, url: String): String? {
+        val req = Request.Builder()
+            .method(method, null)
+            .url(url)
+            .build()
+
+        client.newCall(req).execute().use {
+            val responseBody = it.body?.string()?.split("\n")?.firstOrNull()
```

Comment:
I am fetching a whole HTML page in JavaScript part of the code (`var response = http.request("GET", "https://bb.dom-l.at/");`). To make log message shorter, I take only first line here so that it prints `<html>` instead of the whole downloaded page. This is just a dummy part of code to test JavaScript interoperability anyway, an it will be changed to something more meaningful in the future.